mflike/mflike.py

```diff
@@ -105,7 +105,7 @@ def initialize(self):
         self.log.info("Initialized!")
 
     def get_fg_requirements(self):
-        return {"ells": self.l_bpws,
+        return {"ells": self.l_bpws[:self.lmax_theory + 1],
```

Comment:
I'm always a bit scared when cutting numpy array since I never know what the first indice is. Basically if `self.l_bpws` starts at 2 then I think we have a problem

Reply:
Right, l_bpws should start from 2 if I remember well. So to have it arriving up to lmax_theory we should put `[:self.lmax_theory -1]`